.github/workflows/ibis-ci.yml

```
@@ -22,11 +22,22 @@ jobs:
         uses: chartboost/ruff-action@v1
         with:
           src: './ibis-server'
-          changed-files: 'true'
+          args: 'format --check'
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+          cache: 'maven'
       - name: Start Wren JAVA engine
-        working-directory: ./example/duckdb-tpch-example
+        working-directory: .
         run: |
-          docker compose --env-file .env up -d
+          mkdir etc
+          echo "node.environment=production" >> etc/config.properties
+          echo "wren.directory=./etc/mdl" >> etc/config.properties
```

```suggestion
          echo "wren.directory=./etc/mdl" >> etc/config.properties
          echo "wren.experimental-enable-dynamic-fields=true
" >> etc/config.properties
```
Let's enable dynamic query in default